### PR TITLE
Dev: make changes

### PR DESCRIPTION
**Pull request by McCambley**
Here's a detailed explanation of changes. Here are questios for you both

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -119,6 +119,7 @@ <h2 class="heading heading_section_instagram ">We post interesting stuff on our
             </a>
         </div>
     </section>
+    <section class="fake__footer">Some stuff</section>
```
Comment (larkceresin) [inline index.html:122]: I'm not sure we want this here

File: index.html [hunk 2]
```diff
@@ -119,6 +119,7 @@ <h2 class="heading heading_section_instagram ">We post interesting stuff on our
...
     <footer class="footer">
         <nav class="footer__column footer__column_content_nav">
```
Comment (larkceresin) [inline index.html:124]: On this section ......

**Review by larkceresin — CHANGES_REQUESTED** [review]
Leave a comment here about general stuff

File: index.html (hunk 1 above)
Comment (McCambley) [reply index.html:122]: Oh truuuu thanks

**Pull request closed without merging**